install.sh

```diff
@@ -805,6 +815,14 @@ set_user_subuids_subgids() {
   fi
 }
 
+check_user_existence() {
+  local user="$1"
+  local current_user="${user:-$(get_user)}"
+
+  if ! id "$current_user" &>/dev/null; then
+    abort "User '$current_user' does not exist."
+  fi
+}
 log::info() {
   declare msg="$1"
 
```

Comment:
Check all other places where current user is used for something, like `is_user_in_group()` function, they might need refactoring.